ontology/yaml/resources/METERS/entity_types/WM.yaml

```diff
@@ -26,9 +26,10 @@ WM_STANDARD:
 
 WM_WF:
```

Comment:
this will invalidate the existing type if we add a new required field (existing meters of this type will all be broken). please add the accumulator as optional, or create a new type with both required.

Reply:
got it. thanks sir.